sdk/python/feast/registry_server.py

```diff
@@ -17,6 +17,7 @@
 from feast.permissions.action import CRUD, AuthzedAction
 from feast.permissions.permission import Permission
 from feast.permissions.security_manager import assert_permissions, permitted_resources
+from feast.permissions.server.arrow import inject_user_details_decorator
```

Comment:
I don't think this is correct:
1- This decorator is designed for Arrow Flight APIs, those with parameters starting by `(self, context, ...)`, it would not work for other APIs like these
2- The way it works, for grpc servers, is by using the interceptor defined [here](https://github.com/RHEcosystemAppEng/feast/blob/70667f89df14e1abd78922c30e47e6d8ce184d83/sdk/python/feast/permissions/server/grpc.py#L18) and injected once for all the endpoints [here](https://github.com/RHEcosystemAppEng/feast/blob/70667f89df14e1abd78922c30e47e6d8ce184d83/sdk/python/feast/registry_server.py#L592). The interceptor runs before any endpoint is reached and adds the user credentials to the context.